### ISSUE TITLE
show-file-line command can not show some line of @file xyz.vue

### ISSUE THREAD

**Issue by wangzhaohe** [leo-editor/leo-editor#3010]
Example: When cursor on `// some methods`, run `show-file-line` show nothing in log panel.

Example file: @file xyz.vue

<details>

```html
<!--@+leo-ver=5-thin-->
<!--@+node:swot.20221218204228.2: * @file xyz.vue-->
<!--@@language xml-->
<!--@+others-->
<!--@+node:swot.20221218204228.3: ** template-->
<!--@@language xml-->
<template>

</template>
<!--@+node:swot.20221218204228.4: ** script-->
<!--@@language javascript-->
<!--@delims // -->
<script>
export default {
  name: '',
  //@+<< methods >>
  //@+node:swot.20221218204325.1: *3* << methods >>
  // some methods
  //@-<< methods >>
}
//@delims <!-- --> 
</script>
<!--@@language asciidoc-->
<!--@+doc-->
<!--
-->
<!--@+node:swot.20221218204228.5: ** style-->
<!--@@language css-->
<!--@delims /* */ -->
<style scoped>

/*@delims <!-- --> */
</style>
<!--@@language asciidoc-->
<!--@+doc-->
<!--

-->
<!--@-others-->
<!--@-leo-->
```

</details>

---
Leo Log Window
Leo 6.7.2-devel, None branch, build 3f5198c28f
2022-12-18 20:11:43 +0800
Python 3.9.5, PyQt version 5.15.2
darwin

**Comment by tbpassin** [comment]
I have verified that no line number was displayed for the given example.

**Comment by edreamleo** [comment]
At present, `goto.find_node_start` looks for `<!--` and `--!>` delims in `.vue` files.

It should be possible to look also for `//` delims. I'll make this change only if it can be done cleanly.

**Comment by edreamleo** [comment]
At present, `goto.find_node_start` looks for `<!--` and `--!>` delims in `.vue` files.

It should be possible to look also for `//` delims. I'll make this change only if it can be done cleanly.

**Comment by edreamleo** [comment]
Closed via PR #3012.

**Comment by wangzhaohe** [comment]
Leo Log Window
Leo 6.7.2-devel, devel branch, build b3e28a9800
2022-12-26 15:55:43 -0800
Python 3.9.5, PyQt version 5.15.2
darwin

@edreamleo 
There is still no line number showed in log panel when cursor in << methods >> node after pull request. 

Would you please have a check off the leo/commands/gotoCommands.py file?

**Comment by edreamleo** [comment]
@wangzhaohe Thanks for the testing :-).  I'll take another look.

**Comment by wangzhaohe** [comment]
There is a little change for xyz.vue for instance the comment line should be between `<scrip></script>` or `<style></style>` tag.

```
//@delims <!-- --> 
</script>
...
/*@delims <!-- --> */
</style>
```

**Comment by edreamleo** [comment]
@wangzhaohe I'll take a look soon.

**Comment by edreamleo** [comment]
Closed via PR #3055.
